src/Microsoft.AspNet.SignalR.Client/Transports/TransportHelper.cs

```diff
@@ -57,8 +57,6 @@ public virtual Task<string> GetStartResponse(IHttpClient httpClient, IConnection
 
             var startUrl = UrlBuilder.BuildStart(connection, transport, connectionData);
 
-            httpClient.Initialize(connection);
```

Comment:
I understand the desire to remove the double initialize given the .NET 4.5 version of DefaultHttpClient.

It's more difficult than I first realized to move the call to Initialize outside of TransportHelper. So I would just :shipit: